web/turkey/survey/static/survey/js/auditors/clicks_specific.js

```diff
@@ -0,0 +1,29 @@
+var AuditorClicksSpecifics = {
+    clicks_specifics: [],
+    log_click_content: function (e) {
+        e.stopPropagation();
```

Comment:
I'm not clear on why this call is being made. I'm under the impression this will stop all other handlers on document from receiving click events. Am I wrong? If I'm not, doesn't that mean this is interfering with the operation of other code dependent on click events? If I am, what exactly is this fixing?